pubspec.yaml

```diff
@@ -59,9 +59,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
-  #   - images/a_dot_ham.jpeg
+  assets:
+    - lib/assets/images/imageShoe.jpg
```

Comment:
There is no need to go image by image you can just put folder and flutter will import all images from that folder